spec/slack-ruby-bot/hooks/message_spec.rb

```diff
@@ -31,6 +31,20 @@
     it 'does not return unknown command class' do
       expect(built_in_command_classes).to_not include SlackRubyBot::Commands::Unknown
     end
+    it 'calls the correct command when no-break space present' do
+      client = respond_to?(:client) ? send(:client) : SlackRubyBot::Client.new
+      message_command = SlackRubyBot::Hooks::Message.new
+      allow(client).to(receive(:message)) do |options|
+        @messages ||= []
+        @messages.push(options)
+      end
+
+      run_command = message_command.call(
+        client, Hashie::Mash.new(text: "#{SlackRubyBot.config.user} help", channel: 'channel')
```

Comment:
Because the non-breaking space is not visible here in tests and some editor will clear it, construct a string that uses the unicode character explicitly.